app/models/spree/user.rb

```diff
@@ -14,15 +14,15 @@ class User < ApplicationRecord
            encryptor: 'authlogic_sha512', reconfirmable: true,
            omniauth_providers: [:openid_connect]
 
-    has_many :orders
+    has_many :orders, dependent: nil
     belongs_to :ship_address, class_name: 'Spree::Address'
     belongs_to :bill_address, class_name: 'Spree::Address'
 
     has_and_belongs_to_many :spree_roles,
                             join_table: 'spree_roles_users',
                             class_name: "Spree::Role"
 
-    has_many :spree_orders, class_name: "Spree::Order"
+    has_many :spree_orders, class_name: "Spree::Order", dependent: nil
```

Comment:
I never noticed this. `orders` and `spree_orders` is the same relationship. I'll open a PR to remove the latter...